torch/_dynamo/symbolic_convert.py

```diff
@@ -626,7 +627,9 @@ def get_line_of_code_header(self, lineno=None):
         inline_depth_str = (
             f" (inline depth: {self.inline_depth})" if self.inline_depth > 0 else ""
         )
-        return f"{self.f_code.co_filename}:{lineno} in {self.f_code.co_name}{inline_depth_str}"
+        funcname = get_funcname(self.f_code.co_filename, lineno)
+        funcname_str = "" if funcname is None else f" ({funcname})"
+        return f"{self.f_code.co_filename}:{lineno} in {self.f_code.co_name}{funcname_str}{inline_depth_str}"
```

Comment:
formatting bikeshed: wdyt of `ClassName.foo` instead of `foo (ClassName)`? Former seems more intuitive to me.

Reply:
There are some cases (e.g. in the example above) where the `co_name` does not match up with the last name of the computed `funcname`. So what I'm really printing out for the general case is `foo (ClassName.foo)`.